src/commands/version/version.command.ts

```diff
@@ -0,0 +1,24 @@
+import * as __path from "path";
+import { ICommand } from "..";
+import { IFileService, FileService } from "../../services/file.service";
+
+export class VersionCommand implements ICommand {
+    get name(): string { return "--version"; }
+
+    fileService: IFileService = new FileService();
+
+    run = (): Promise<void> => {
+        let path = __path.join(__dirname, '../../..', 'package.json');
+        return this.getPackageFile(path)
+        .then(rslt => {            
+            console.log(`shaman-cli version ${rslt.version}`);
```

Comment:
This is nit-picky (and technically unnecessary), but figured it would be good practice for resolving PR comments. Lets improve the format of the output string, something like:

```ts
console.log(`shaman-cli version: v${rslt.version}`);
```

Reply:
Updated the version output string to be more clear.